src/HttpResponsePacket.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2021 Pei-Ru Wang, Cheryl Huang, Yuan Zhou. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/**
+ * Project MyHttpServer
+ * @author Pei-Ru Wang, Cheryl Huang, Yuan Zhou
+ * @version 0.0.1
+ */
+
+#include "HttpResponsePacket.h"
+
+/**
+ * HttpResponsePacket implementation
+ */
+
+namespace status_strings {
+
+const std::string ok = "HTTP/1.0 200 OK\r\n";
```

Comment:
Please follow constant naming style:
https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
[Optional] Use constexpr char[] instead of std::string to avoid constructor.